src/realm/impl/transact_log.hpp

```diff
@@ -2586,10 +2587,10 @@ class TransactReverser {
         return true; // No-op
     }
 
-    bool clear_table()
+    bool clear_table(size_t old_size)
     {
-        // FIXME: Add a comment on why we call insert_empty_rows() inside clear_table()
```

Comment:
I'm not sure what should be explained? Inserting rows into the table is the inverse operation of clearing the table, so it seems like the obvious thing to be doing here.